### DISCUSSION TITLE
Can this be used in Linux?

### DISCUSSION THREAD

**Discussion by DevInfinix** [Akascape/py-window-styles#14] [General]
DE: KDE Dr460nized (Wayland)
Distro: Garuda Linux (Arch based)
Theme: Vivid Dark Icons

![image](https://github.com/Akascape/py-window-styles/assets/72700816/244fd114-b339-439b-839b-8e6b20c48f96)

**Comment by Akascape** [comment]
No, it is only for windows. That's why it is py **windows** style.
